src/tests/end2end/ScissorTests.cpp

```diff
@@ -98,86 +94,76 @@ TEST_P(ScissorTest, EmptyRect) {
         return;
     }
 
-    utils::BasicFramebuffer fb = utils::CreateBasicFramebuffer(device, 2, 2);
-    nxt::RenderPipeline pipeline = CreateQuadPipeline(fb.renderPass);
+    utils::BasicRenderPass renderPass = utils::CreateBasicRenderPass(device, 100, 100);
```

Comment:
This test changed size, is that intentional?

Reply:
No, done